src/opnsense/service/templates/OPNsense/Monit/monitrc

```diff
@@ -135,7 +135,11 @@ check {{ service.type }} {{ service.name }} {{ path }}
 {%      endfor %}
 {%     endif %}
 {%     if service.start|default('') != '' %}
-   start program = "{{ service.start }}"
+{%       set srv_start_timeout = '' %}
+{%       if service.starttimeout|default('') != '' %}
+{%         set srv_start_timeout = "with timeout " ~ service.starttimeout ~ " seconds" %}
+{%       endif %}
+   start program = "{{ service.start }}" {{ srv_start_timeout }}
```

Comment:
If we are always going to set the timeout, it might be better to replace the above with:

```
start program = "{{ service.start }}" with timeout {{service.starttimeout|default('30')}} seconds
```

As timeout is 30 seconds by default anyway, we should be safe here.

Reply:
I've made your recommended change. 
I hesitated doing it the way you recommended because it would hard code Monit's default value in Opnsense's source code and if it were to change, it would also need to change in Opnsense.  On the other hand, its less code which is nice.

Thanks for the feedback.

Reply:
@spoutin setting the default in the model would lead to the same effect as it persists the "30" after a save. In this case I don't think it matters that much to force it to default.